internal/pkg/model/object.go

```diff
@@ -110,7 +114,7 @@ type Config struct {
 	Description       string                 `json:"description" diff:"true" descriptionFile:"true"`
 	ChangeDescription string                 `json:"changeDescription"`
 	Content           *orderedmap.OrderedMap `json:"configuration" validate:"required" diff:"true" configFile:"true"`
-	Blocks            Blocks                 `json:"-" validate:"dive"` // loaded transformation's blocks, filled in only for the LOCAL state
```

Comment:
`Config.Blocks` su taky kostlivec, ktory nejak fungoval, ale nebol spraveny poriadne.
```
 // loaded transformation's blocks, filled in only for the LOCAL state
```

Transformation blocks su na disku reprezentovane ako adresar `blocks`, v API su sucastou konfiguracie, pod klucom `blocks`:
https://github.com/keboola/keboola-as-code/tree/main/test/pull/out/bar/transformation/keboola.python-transformation-v2/python-transformation

**Fungovalo to tak:**
- pri local load 
  - sa nacitali `blocks` z adresara a ulozili sa priamo do `config.Content` pod kluc `blocks` ako `orderedMap`.
  - zaroven sa ulozili do `Config.Blocks` ako `[]*Block`, aby sme pri `rename` vedeli, aka blocks lokalne existuju.
  - tj `local state -> api state`
- pri remote load
  - sa nedialo nic
  - diff `blocks` sa robil vramci `Config.Content`

**Upravil som to:**
- local load nacita adresar `blocks` a ulozi ho do `Config.Blocks` (internal representation)
- remote load nacita `blocks` z `Config.Content` a ulozi do `Config.Blocks` (internal representation)
- diff sa robi nad `Config.Blocks`
- a save remote/local funguje ako load len opacne